prow/cmd/peribolos/main.go

```diff
@@ -812,6 +841,50 @@ func configureOrg(opt options, client github.Client, orgName string, orgConfig o
 	return nil
 }
 
+type repoClient interface {
+	GetRepos(orgName string, isUser bool) ([]github.Repo, error)
+	CreateRepo(owner string, isUser bool, repo github.Repo) (*github.Repo, error)
+}
+
+func toGitHubRepo(name string, definition org.Repo) github.Repo {
+	ghRepo := github.Repo{Name: name}
+
+	updateString(&ghRepo.Description, definition.Description)
+	updateString(&ghRepo.Homepage, definition.HomePage)
+	updateBool(&ghRepo.Private, definition.Private)
+	updateBool(&ghRepo.HasWiki, definition.HasWiki)
+	updateBool(&ghRepo.HasProjects, definition.HasProjects)
+	updateBool(&ghRepo.HasIssues, definition.HasIssues)
+
+	return ghRepo
+}
+
+func configureRepos(client repoClient, orgName string, orgConfig org.Config) error {
```

Comment:
The comments above said "update", too -- where do we do that?

Reply:
In my commit buffer - I want to polish it more (also based on feedback from this PR) and add the feature in another PR. I've cleaned up the comments/help strings for now.